deployment/eclipse-kuksa/deploy_traccar.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+
+######################################################################
+# Copyright (c) 2019 Bosch Software Innovations GmbH [and others]
+#
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+########################################################################
+
+# Make the script fail if a command fails
+set -e
+
+SCRIPTPATH=$(dirname "$(readlink -f "$0")")
+
+NAMESPACE=kuksa
+DOCKER_IMAGE_NAME="traccar/traccar"
+VERSION="4.5"
+
+TARGET_DIR=$SCRIPTPATH/traccar/target
+DEPLOYMENT_DESCRIPTOR_TEMPLATE=$SCRIPTPATH/traccar/traccar-deployment.yaml
+DEPLOYMENT_DESCRIPTOR=$TARGET_DIR/traccar-deployment.yaml
+SERVICE_DESCRIPTOR_TEMPLATE=$SCRIPTPATH/traccar/traccar-service.yaml
+SERVICE_DESCRIPTOR=$TARGET_DIR/traccar-service.yaml
+
+. $SCRIPTPATH/../kubernetes/includes/allIncludes.inc
+
```

Comment:
The 'kubernetes' folder does not contain the 'includes' folder yet.  So we need to merge the respective PR first.

Reply:
seems to be part of #81 or am I missing changes introduced by other PRs?

Reply:
Yes, this PR depends on #81 that moves the files to the kubernetes folder.